src/infra/alarm/template.yaml

```diff
@@ -148,9 +160,247 @@ Resources:
       Namespace: AWS/SQS
       Period: 60
       Statistic: Maximum
+
+  AccountDeletionProcessorQueueProcessingSlowly:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSLowAlertNotificationTopicARN
+      AlarmDescription: !Sub "Account Deletion Processor Queue processing slowly. ${RunBookURL}"
+      ComparisonOperator: GreaterThanThreshold
+      Threshold: 60
+      EvaluationPeriods: 1
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: QueueName
+          Value: !Ref AccountDeletionProcessorQueueName
+      MetricName: ApproximateAgeOfOldestMessage
+      Namespace: AWS/SQS
+      Period: 60
+      Statistic: Maximum
+
+  AccountDeletionProcessorQueueProcessingTooSlow:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSLowAlertNotificationTopicARN
+      AlarmDescription: !Sub "Account Deletion Processor Queue processing too slow. ${RunBookURL}"
+      ComparisonOperator: GreaterThanThreshold
+      Threshold: 600
+      EvaluationPeriods: 1
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: QueueName
+          Value: !Ref AccountDeletionProcessorDLQName
+      MetricName: ApproximateAgeOfOldestMessage
+      Namespace: AWS/SQS
+      Period: 60
+      Statistic: Maximum
+
+  AccountDeletionProcessorDLQProcessingSlowly:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSLowAlertNotificationTopicARN
+      AlarmDescription: !Sub "Account Deletion Processor DLQ processing slowly. ${RunBookURL}"
```

Comment:
I wouldn't say this is processing slowly, but we have at least one message in our DLQ which has been there for longer than a minute.

Reply:
Shall we keep this one as a Low with a threshold of 1 with metric being the number of messages, and get rid of AccountDeletionProcessorDLQProcessingTooSlow (high severity one)

Reply:
So  AccountDeletionProcessorDLQHasBacklog alarm with a threshold of 1 and ApproximateNumberOfMessagesVisible metric, but then change to a high alert or keep to low if just having one alarm in place for that?

Reply:
Yes to above, and we keep it Low for reason Taher mentioned in dev techy